lisa/util/shell.py

```diff
@@ -285,6 +291,9 @@ def is_connected(self) -> bool:
             is_inner_shell_ready = True
         return is_inner_shell_ready
 
+    def set_sudo_required_password(self, value: Optional[bool]) -> None:
```

Comment:
It looks the value doesn't need to be None again, so the declaration is like, `def set_sudo_required_password(self, value bool) -> None:`

Reply:
Done